packaging/hudi-spark-bundle/pom.xml

```diff
@@ -95,6 +95,7 @@
                   <include>org.apache.httpcomponents:fluent-hc</include>
                   <include>org.antlr:stringtemplate</include>
                   <include>org.apache.parquet:parquet-avro</include>
+                  <include>org.apache.avro:avro</include>
```

Comment:
Can you also shade Avro in other remaining bundles?

Reply:
I have check engine bundle:  hudi-flink-bundle, hudi-spark-bundle, hudi-presto-bundle, hudi-trino-bundle, hudi-hadoop-mr-bundle, for others moudle, I don't know if need shade it.